DuckDuckGo/OnboardingExperiment/ProgressBarView.swift

```diff
@@ -144,7 +144,7 @@ struct ProgressBarGradient: View {
                     let nextStep = stepInfo.currentStep < stepInfo.totalSteps ? stepInfo.currentStep + 1 : 1
                     stepInfo = OnboardingProgressIndicator.StepInfo(currentStep: nextStep, totalSteps: stepInfo.totalSteps)
                 }, label: {
-                    Text("Update Progress")
+                    Text(verbatim: "Update Progress")
```

Comment:
Why did this change?  `verbatim` means everyone will see "Update Progress" but shouldn't it be localised?

Reply:
Ah this is just a SwiftUI preview I built to test the behaviour of the progress bar. When I tap that button I simulate the progress bar increasing its progress. At translation time I noticed it was translating those strings.